### DISCUSSION TITLE
[Question][Lecture material]

### DISCUSSION THREAD

**Discussion by tanapthetimid** [prosyslab-classroom/cs348-information-security#701] [Q&A]
Name: Tanapoom Sermchaiwong

I wonder if there are some materials/textbook/resources I can read in place of next week's lectures, since I will not be able to attend class for a personal reason.

**Comment by KihongHeo** [comment]
Sorry to hear that you cannot attend the next classes. (Actually, there is your contribution to the course materials.)

The lectures will be largely based on the [Web security chapter](https://people.scs.carleton.ca/~paulv/toolsjewels/TJrev1/ch9-rev1.pdf) of the textbook. Please take a look.

**Comment by tanapthetimid** [comment]
Thank you very much! It is regrettable to have to miss next week.
